sdk/Sdk/Tasks/ZipDeploy/ZipDeployTask.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.NET.Sdk.Functions.Http;
+using Microsoft.NET.Sdk.Functions.MSBuild.Tasks;
+
+// IMPORTANT: Do not modify this file directly with major changes
+// This file is a copy from this project (with minor updates) -- https://github.com/Azure/azure-functions-vs-build-sdk/blob/b0e54a832a92119e00a2b1796258fcf88e0d6109/src/Microsoft.NET.Sdk.Functions.MSBuild/Microsoft.NET.Sdk.Functions.MSBuild.csproj
+// Please make any changes upstream first.
+
+namespace Microsoft.NET.Sdk.Functions.Tasks
+{
+#if NET472
+    [LoadInSeparateAppDomain]
+    public class ZipDeployTask : AppDomainIsolatedTask
+#else
+    public class ZipDeployTask : Task
+#endif
+    {
+        private const string UserAgentName = "functions-core-tools";
```

Comment:
Again, I know this is a problem with the original implementation, but do we want this to report `functions-core-tools`? That's not really accurate.